Tasks/AzureCloudPowerShellDeployment/task.json

```diff
@@ -217,6 +217,8 @@
         "Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "Found more than one file to deploy with search pattern {0}. There can be only one.",
         "Storagekeysaredefinedininvalidformat": "Storage keys are defined in invalid format.",
         "Unabletofind0usingprovidedsubscription": "Unable to find {0} using provided subscription",
-        "Currentversionofazurepowershelldontsupportexternalstorageaccountforconfiguringdiagnostics": "Current version of azure powershell don't support external storage account for configuring diagnostics."
+        "Currentversionofazurepowershelldontsupportexternalstorageaccountforconfiguringdiagnostics": "Current version of azure powershell don't support external storage account for configuring diagnostics.",
```

Comment:
.... azure powershell **does not** support....

Reply:
+1